modules/social_features/social_featured_items/social_featured_items.install

```diff
@@ -18,3 +18,17 @@ function social_featured_items_update_8001() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Add translations support for "Featured Item(s)" custom block fields.
+ */
+function social_featured_items_update_10200() {
```

Comment:
Since we add this to 10.3 milestone, need to rename the number to start with 103

Reply:
fixed